OrbitCore/CoreUtils.h

```diff
@@ -231,7 +231,7 @@ inline void fopen_s(FILE** fp, const char* fileName, const char* mode) {
 }
 #endif
 
-namespace OrbitUtils {
+namespace orbit_utils {
```

Comment:
I would just make this `orbit_core`, too.

Reply:
I'm missing a lot of context :-/. I'm happy to do a second pass on these suggestions, but sadly I couldn't really arrive at them myself at this time.

Reply:
I have a new commit ready with this change, will update the PR in a bit.